frontend/src/api/evaluationApi.js

```diff
@@ -12,7 +12,13 @@ const EvaluationApi = {
   sendSelfEvaluationResults: formResults => api.post("/form", formResults),
 
   sendEvaluationResults: (formResults, id) =>
-    api.post("/form", formResults, { params: { id } })
+    api.post("/form", formResults, { params: { id } }),
+
+  getEvaluationResults: async userId => {
+    const { data } = await api.get(`form?id=${userId}`);
```

Comment:
```suggestion
    const { data } = await api.get(`/form?id=${userId}`);
```
It's a bit more clear this way